### PR TITLE
merge to powerplatform

### PR DESCRIPTION
**Pull request by varunynoox**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json [hunk 1]
```diff
@@ -4075,7 +4075,7 @@
         "operationId": "GenerateDocumentSingle_V1",
         "x-ms-api-annotation": {
           "family": "GenerateDocumentSingle",
-          "status": "Preview",
+          "status": "Production",
```
Comment (vmanoharas) [inline certified-connectors/PDF4me Connect/apiDefinition.swagger.json:4178]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json [hunk 2]
```diff
@@ -4198,7 +4198,7 @@
         "operationId": "GenerateDocumentMultiple_V1",
         "x-ms-api-annotation": {
           "family": "GenerateDocumentMultiple",
-          "status": "Preview",
+          "status": "Production",
```
Comment (vmanoharas) [inline certified-connectors/PDF4me Connect/apiDefinition.swagger.json:4301]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json [hunk 3]
```diff
@@ -5024,7 +5024,7 @@
         "x-ms-api-annotation": {
           "family": "ClassifyDocument",
           "revision": 1,
-          "status": "Preview"
+          "status": "Production"
```
Comment (vmanoharas) [inline certified-connectors/PDF4me Connect/apiDefinition.swagger.json:5127]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json [hunk 4]
```diff
@@ -5094,7 +5094,7 @@
         "x-ms-api-annotation": {
           "family": "ParseDocument",
           "revision": 1,
-          "status": "Preview"
+          "status": "Production"
```
Comment (vmanoharas) [inline certified-connectors/PDF4me Connect/apiDefinition.swagger.json:5197]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json [hunk 5]
```diff
@@ -261,7 +261,7 @@
         "x-ms-api-annotation": {
           "family": "CreateSwissQrBill",
           "revision": 2,
-          "status": "Preview"
+          "status": "Production"
```
Comment (vmanoharas) [inline certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:264]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json [hunk 6]
```diff
@@ -336,7 +336,7 @@
         "x-ms-api-annotation": {
           "family": "ReadSwissQrBill",
           "revision": 2,
-          "status": "Preview"
+          "status": "Production"
```
Comment (vmanoharas) [inline certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:339]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json [hunk 7]
```diff
@@ -486,7 +486,7 @@
         "x-ms-api-annotation": {
           "family": "SplitDocBySwissQrCode",
           "revision": 2,
-          "status": "Preview"
+          "status": "Production"
```
Comment (vmanoharas) [inline certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:489]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me/apiDefinition.swagger.json [hunk 8]
```diff
@@ -3904,7 +3981,7 @@
         "operationId": "GenerateDocumentSingle_V1",
         "x-ms-api-annotation": {
           "family": "GenerateDocumentSingle",
-          "status": "Preview",
+          "status": "Production",
```
Comment (vmanoharas) [inline certified-connectors/PDF4me/apiDefinition.swagger.json:4084]: Can you please let's know the reason for changing from Preview and Production.

File: certified-connectors/PDF4me/apiDefinition.swagger.json [hunk 9]
```diff
@@ -4027,7 +4104,7 @@
         "operationId": "GenerateDocumentMultiple_V1",
         "x-ms-api-annotation": {
           "family": "GenerateDocumentMultiple",
-          "status": "Preview",
+          "status": "Production",
```
Comment (vmanoharas) [inline certified-connectors/PDF4me/apiDefinition.swagger.json:4207]: Can you please let's know the reason for changing from Preview and Production.

**Review by vmanoharas — COMMENTED** [review]
Can you please let's know the reason for changing from Preview and Production. This will be taken care by us when moving your connector to GA.

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 1 above)
Comment (varunynoox) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:4178]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 2 above)
Comment (varunynoox) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:4301]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 3 above)
Comment (varunynoox) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:5127]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 4 above)
Comment (varunynoox) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:5197]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json (hunk 5 above)
Comment (varunynoox) [reply certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:264]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json (hunk 6 above)
Comment (varunynoox) [reply certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:339]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json (hunk 7 above)
Comment (varunynoox) [reply certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:489]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me/apiDefinition.swagger.json (hunk 8 above)
Comment (varunynoox) [reply certified-connectors/PDF4me/apiDefinition.swagger.json:4084]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me/apiDefinition.swagger.json (hunk 9 above)
Comment (varunynoox) [reply certified-connectors/PDF4me/apiDefinition.swagger.json:4207]: the connector is now available under GA. the action has been tested and should be moved to Production status

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 1 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:4178]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meconnect/

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 2 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:4301]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meconnect/

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 3 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:5127]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meconnect/

File: certified-connectors/PDF4me Connect/apiDefinition.swagger.json (hunk 4 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me Connect/apiDefinition.swagger.json:5197]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meconnect/

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json (hunk 5 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:264]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meswissqr/

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json (hunk 6 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:339]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meswissqr/

File: certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json (hunk 7 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me SwissQR/apiDefinition.swagger.json:489]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meswissqr/

File: certified-connectors/PDF4me/apiDefinition.swagger.json (hunk 8 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me/apiDefinition.swagger.json:4084]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4me/

File: certified-connectors/PDF4me/apiDefinition.swagger.json (hunk 9 above)
Comment (vmanoharas) [reply certified-connectors/PDF4me/apiDefinition.swagger.json:4207]: Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4me/

**Comment by vmanoharas** [comment]
Hello Partner, 

All the connectors have already been transitioned from Preview to Production and are now in the GA (Generally Available) stage. Additionally, please consult the provided links to their respective documentation, where you will notice that there is no longer a "Preview" tag. This indicates that the required changes have been successfully implemented in the code.  Please discard this PR or let me know if you have any questions.

**Comment by varunynoox** [comment]
but the actions still have Preview tag in them

image

**Comment by vmanoharas** [comment]
Hello @varunynoox 
Thank you very much for notifying this, could you please modify only actions and re-submit it.  We will process your request. Apologize for the inconvenience caused.

**Comment by varunynoox** [comment]
I have changes in action and have commit them. Please verify and merge the pull request

**Review by vmanoharas — APPROVED** [review]
Hello @varunynoox 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**